src/main/java/se/kth/sda/skeleton/posts/PostService.java

```diff
@@ -1,7 +1,47 @@
 package se.kth.sda.skeleton.posts;
```

Comment:
At some point, we should probably update the folder names. Perhaps something for the backlog.

Reply:
yap...was looking at that